scripts/internal/manylinux-build-module-wheels.sh

```diff
@@ -101,6 +101,8 @@ for PYBIN in "${PYBINARIES[@]}"; do
       -DPython3_INCLUDE_DIR:PATH=${Python3_INCLUDE_DIR} \
       ${CMAKE_OPTIONS} \
     || exit 1
+    # Copy generated headers into sources before cleaning
+    find _skbuild -type f -wholename "**/cmake-build/include/*" -print -exec cp {} include \;
     ${PYBIN}/python setup.py clean
```

Comment:
@LucasGandel thanks for the patch!

Can we just remove this line instead?

Reply:
@thewtex Thanks, I just edited the change. You are right, it's better to remove the clean step and let the module CI decide what it wants to pick from this build.

Reply:
@LucasGandel thanks!